README.md

```diff
@@ -0,0 +1,5 @@
+# FCC Test Suite Template
```

Comment:
Be careful, you should remove this comment

Reply:
I think we can make this comment better by making improvements to how we communicate to the user😉 Remember that from the previous lesson on ["How to give a code review"](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md) we were taught to;
- be suggestive in our mode of communication rather than being instructive or giving orders😉
<img width="866" alt="no_orders_cr" src="https://user-images.githubusercontent.com/64643995/212743525-df599560-8131-461c-8c7d-05afc6b65d09.png">
- give descriptions as to why the student should make the change you're requesting😉
<img width="581" alt="explain-crr" src="https://user-images.githubusercontent.com/64643995/212743689-7ee5a271-c17a-4d2b-a4b2-8d8e8c8d36d4.png">

kindly edit your comments and make changes with the guide lines given above and come up with a more descriptive comment😉🤗